app/api/services/briefs.py

```diff
@@ -153,6 +154,13 @@ def get_briefs_by_filters(self, status=None, open_to=None, brief_type=None, loca
 
                 cond = or_(lot_cond, ids_cond, aoe_cond)
                 query = query.filter(cond)
+            elif 'atm' in brief_type_filters:
+                # this is a list of historic prod brief ids we want to show when the atm filter is active
+                atm_ids = [136, 180, 351, 207, 383, 453, 485, 490, 548, 568, 633, 743, 819, 830, 862, 975, 1071,
```

Comment:
Not sure if this was supposed to be sorted but it was oh so close!
```suggestion
                atm_ids = [136, 180, 207, 351, 383, 453, 485, 490, 548, 568, 633, 743, 819, 830, 862, 975, 1071,
```